### DISCUSSION TITLE
Biomarkers of mychoice

### DISCUSSION THREAD

**Discussion by sharm842** [dayoonkwon/BioAge#29] [Q&A] [answered]
Hi, 

First of all, I am an intermediate self-taught R user. I am trying to make three metrics of bio age with the variables of my choice from the dataset I have called Bio_v1n" it only has WBC, MCV, RDW, Lymphocyte, Creatinin, total cholesterol, and HbA1c. Could anyone clarify which part of the code should only be changed for these biomarkers? Secondly, do I have to train and project with the NHANES-3&4 dataset?

**Comment by dayoonkwon** [comment, marked as answer]
As an intermediate R user, you’ll find that the BioAge README file provides comprehensive guidance on how to adapt the code for custom biomarkers. Specifically, look for the sections detailing how to specify biomarkers for analysis; in your case, you’ll replace the default biomarker list with your own selection (WBC, MCV, RDW, Lymphocyte, Creatinin, total cholesterol, HbA1c).

Yes, you should train the algorithm and then project it onto your "Bio_v1n" dataset.
